packages/@aws-cdk/aws-eks/README.md

```diff
@@ -564,7 +564,9 @@ For example:
 ```ts
 declare const cluster: eks.Cluster;
 const manifest = cluster.addManifest('manifest', {/* ... */});
-manifest.node.addDependency(cluster.albController ?? [])
```

Comment:
@rix0rrr This was just introduced in https://github.com/aws/aws-cdk/pull/17618, and was a new example (so no cache issues). This error should have been reported by the `strict` mode enforced on `eks` if that were working properly. It was also not reported in the build failure on the forward-merge PR, but was reported when I ran `yarn rosetta:extract --compile` locally.

Reply:
Well shoot!